ios/Video/RCTVideo.swift

```diff
@@ -192,6 +199,14 @@ class RCTVideo: UIView, RCTVideoPlayerViewControllerDelegate, RCTPlayerObserverH
         _player?.pause()
         _player?.rate = 0.0
     }
+    
+    @objc func applicationDidBecomeActive(notification: NSNotification!) {
+        if _playInBackground || _playWhenInactive || _paused { return }
+
+        // Resume the player or any other tasks that should continue when the app becomes active.
+        _player?.play()
+        _player?.rate = 1.0
```

Comment:
Let's restore playback rate when resuming video
```suggestion
        _player?.rate = _rate
```